### PR TITLE
0.2.x: Handle when model value is too big for message

### PR DESCRIPTION
**Pull request by sverhoeven**
Refs #76

### CODE REVIEW

File: test/test_subproc.py [hunk 1]
```diff
@@ -139,6 +141,18 @@ def test_get_var_values():
     del client
 
 
+def test_get_value_huge():
```
[the remark this answers is not in the document]
Comment (nielsdrost) [reply test/test_subproc.py:144]: See #80

File: grpc4bmi/utils.py [hunk 2]
```diff
@@ -36,3 +36,7 @@ def stage_config_file(filename, input_dir, input_mount_point, home_mounted=False
         # Assume filename exists inside container or model does not need a file to initialize
         pass
     return fn
+
+
+# grpc max message size is 4Mb
+GRPC_MAX_MESSAGE_LENGTH = 4 * 1024 * 1024
```
[the remark this answers is not in the document]
Comment (sverhoeven) [reply grpc4bmi/utils.py:42]: I looked for it, it is set in https://github.com/grpc/grpc/blob/v1.31.x/include/grpc/impl/codegen/grpc_types.h#L464 , but could not get it out.

File: test/fake_models.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+    def set_value_at_indices(self, name, inds, src):
+        self.value[inds] = src
+
+
+class HugeModel(DTypeModel):
+    """Model which has value which does not fit in message body
+    Can be run from command line with
+    ..code-block:: bash
+        run-bmi-server --path $PWD/test --name fake_models.HugeModel --port 55555 --debug
```
[the remark this answers is not in the document]
Comment (sverhoeven) [reply test/fake_models.py:185]: Backport problem, `--debug` is available on master branch, but not on 0.2.x branch. Will remove

File: test/test_subproc.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sverhoeven) [reply test/test_subproc.py:144]: indeed it is

File: grpc4bmi/utils.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nielsdrost) [reply grpc4bmi/utils.py:42]: Perhaps add the location as a comment and leave it at that?

[the remark this answers is not in the document]
Comment (sverhoeven) [reply grpc4bmi/utils.py:42]: Done in https://github.com/eWaterCycle/grpc4bmi/commit/053cc4c786044d626410aabc3a1d21c4656e599f

**Pull request merged**